### DISCUSSION TITLE
在系统中使用30分钟不等于180s，博主检查说明和参数是否有错

### DISCUSSION THREAD

**Discussion by wofute** [jeessy2/ddns-go#1180] [General]
每 10 秒检查一次本地 IP 变化， 每 30 分钟对比一下 IP 变化， 实现 IP 变化即时触发更新且不会被服务商限流， 如果使用接口获取IP， 需要注意接口限流
./ddns-go -s install -f 10 -cacheTimes 180

**Comment by jeessy2** [comment]
cacheTimes是次数。 180*10秒=1800秒，刚好30分钟
